firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerformanceInitializer.java

```diff
@@ -15,17 +15,23 @@
 package com.google.firebase.perf;
 
 import com.google.firebase.perf.application.AppStateMonitor;
+import com.google.firebase.perf.logging.AndroidLogger;
 
 /**
  * FirebasePerformanceInitializer to initialize FirebasePerformance during app cold start
  *
  * @hide
  */
 public final class FirebasePerformanceInitializer implements AppStateMonitor.AppColdStartCallback {
+  private static final AndroidLogger logger = AndroidLogger.getInstance();
 
   @Override
   public void onAppColdStart() {
     // Initialize FirebasePerformance when app cold starts.
-    FirebasePerformance.getInstance();
+    try {
+      FirebasePerformance.getInstance();
+    } catch (IllegalStateException ex) {
+      logger.info("Failed to get instance of FirebasePerformance: %s", ex);
```

Comment:
Do we want to give some more verbose message here? Eg: `Firebase Performance not initialized. We would not be collecting any performance metrics. Make sure FirebaseApp is initialized.`